docs/setup_guides/hpc.rst

```diff
@@ -109,10 +111,114 @@ Copy that string into your ``jupyter_notebook_config.py`` config file
 
     c.NotebookApp.password = u'sha1:69a76df803b9:99ca27341563cd85ba4e78684128e1f4ad2d8d0d'
 
-Great. We're done with that.
+For security reasons, we recommend making sure your ``jupyter_notebook_config.py``
+is readable only by you. For more information on and other methods for
+securing Jupyter, check out
+`Securing a notebook server <http://jupyter-notebook.readthedocs.io/en/stable/public_server.html#securing-a-notebook-server>`__
+in the Jupyter documentation.
+
+------------
+
+From here, we have two options. Option 1 will start a Jupyter Notebook Server
+and manage dask and using the `dask-jobqueue`_ package. Option 2 will start a dask
+cluster using `dask-mpi` and will run a Jupyter server as part of the dask cluster.
+
+Deploy Option 1: Jupyter + dask-jobqueue
+----------------------------------------
+
+Start a Jupyter Notebook Server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now that we have Jupyter configured, we can start a notebook server. In many
+cases, your system administrators will want you to run this notebook server in
+an interactive session on a compute node. This is not universal rule, but it is
+one we'll follow for this tutorial.
+
+In our case, the Cheyenne super computer uses the PBS job scheduler, so typing:
+
+::
+
+    (pangeo) $ qsub -I -l select=1:ncpus=4 -l walltime=03:00:00 -q regular
+
+This will get us an interactive job on the `regular` queue for three hours. You
+may not see the `pangeo` environment anymore in your prompt, in this case, you
+will want to reactivate it.
```

Comment:
Are interactive jobs like this standard across HPC systems?

Reply:
Depends what you mean by "like" and what you mean by "standard". Certainly almost all HPC systems in different places have some sort of interactive jobs somewhat similar to these. The details on how to start them, and where you can run them, and what environment they have is different from site to site.

Reply:
In my experience, almost all HPC systems have some way to get users interactive sessions on compute nodes. This helps get users off the log in nodes and provides users with an interactive environment to test jobs with the same environment that the batch queues get.